x-pack/plugin/searchable-snapshots/src/main/java/org/elasticsearch/xpack/searchablesnapshots/cache/blob/BlobStoreCacheMaintenanceService.java

```diff
@@ -459,69 +463,72 @@ public void onFailure(Exception e) {
 
                 if (searchResponse == null) {
                     final SearchSourceBuilder searchSource = new SearchSourceBuilder();
+                    searchSource.fetchField(CachedBlob.CREATION_TIME_FIELD);
```

Comment:
Can we use a `new FieldAndFormat(CachedBlob.CREATION_TIME_FIELD, "epoch_millis")` to avoid parsing the string to long in `getCreationTime`?

Reply:
Yes, but it seems that the Fields API still return a String to be parsed.